Resources/ServerInfo/Guidebook/_RMC14/Rules/Overview.xml

```diff
@@ -1,22 +1,20 @@
 ﻿<Document>
-  #RMC-14 Rules
+  # RMC-14 Rules
 
   If you are unable to click any of the links, use the in-game guidebook version.
 
-  ##Recent Changes
+  ## Recent Changes
   Read the relevant sections for the entire text.
-  - (Marine roleplay, metagaming) Surrounding a comms tower entirely with reinforced walls to prevent Xenos attacking it without T3s is not allowed.
-  - (Survivor roleplay) Survivors should not be rushing to attack the hive before the roundstart xenos have had a chance to evolve (5 minutes).
-  - [bold]Hunting or Securing Monkeys to deny Xenonid captures is considered Powergaming.[/bold]
-  - (End-of-round grief, Zero tolerance) Suicides done at round-end without a valid roleplay reason are now prohibited and are counted as regular end-of-round grief, and will result in an immediate unappealable 3-hour ban.
-  - (General Roleplay Standards) Suicides can be done with a valid roleplay reason. Read the section for more details. An example is suiciding as a survivor in a last stand scenario, when it is clear that you will be captured by xenonids and there is no hope of fighting back.
+  - (Command/MP Roleplay Standards) You are expected to be able to communicate clearly when giving orders or acting on things related to marine law, like arrest handling and appeals.
+  - (General Roleplay Standards, metagaming) Surrounding a comms tower entirely with reinforced walls to prevent Xenos attacking it without T3s is not allowed.
+  - (General Roleplay Standards, Suicides) Suicides can be done with a valid roleplay reason. Read the section for more details. An example is suiciding as a survivor in a last stand scenario, when it is clear that you will be captured by xenonids and there is no hope of fighting back. Do not attempt then cancel a suicide, it is treated the same.
```

Comment:
"Do not attempt then cancel a suicide, it is treated the same."
Change "then" to 'to'.